src/wp-admin/includes/class-wp-debug-data.php

```diff
@@ -64,6 +64,16 @@ public static function debug_data() {
 		// Set up the array that holds all debug information.
 		$info = array();
 
+		$info['classicpress'] = array(
+			'label'  => __( 'ClassicPress' ),
+			'fields' => array(
+				'platform' => array(
+					'label' => __( '** Platform **' ),
+					'value' => $core_version,
+				),
+			),
+		);
+
```

Comment:
This creates a duplicate "ClassicPress" tab under Info tab:
![image](https://github.com/ClassicPress/ClassicPress-v2/assets/1692600/76b4c9d4-cb7f-43ef-abf7-62df75ef51b3)

Would it be easier to simply add a heading where copied data is generated?
https://github.com/ClassicPress/ClassicPress-v2/blob/ebfcc7b7fa8c96ce7a49c53679a28e43d73df432/src/wp-admin/includes/class-wp-debug-data.php#L1487

Reply:
New change looks good. 
![image](https://github.com/ClassicPress/ClassicPress-v2/assets/1692600/63fe3bc7-f78c-44a8-85c9-26c9b653e9ff)